doc/openapi-user-guide.md

```diff
@@ -194,7 +194,14 @@ spec:
   * Only first `servers[0].url` element in `servers` list parsed as *private base url*. As OpenAPI specification `basePath` property, `servers[0].url` URL's base path component will be used.
   * `servers` element in path item or operation items are not supported.
   * Just a single top level security requirement supported. Operation level security requirements not supported.
-  * Supported security schemes: `apiKey`.
+  * Supported security schemes: `apiKey`. 
+* As detailed in section [OpenID Connect and OAuth2 example](#openid-connect-and-oauth2-example)  - 
```

Comment:
add text below to a separated sub-section with title. Something like `openIdConnect/oauth2 limitations`

Reply:
done. Thank you